specification/sql/resource-manager/readme.md

```diff
@@ -24,13 +24,23 @@ These are the global settings for the Sql API.
 title: SqlManagementClient
 description: 'The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables you to create, retrieve, update, and delete databases.'
 openapi-type: arm
-tag: package-composite-v3
+tag: package-preview-2020-02
```

Comment:
This should always point to composite package. if you want your APIs included, add them to the composite package.

Reply:
Modified. Please check.